python/ribasim/tests/test_io.py

```diff
@@ -27,6 +27,11 @@ def __assert_equal(a: DataFrame, b: DataFrame, ignore_metas: bool = True) -> Non
         a["time"] = a.time.astype("datetime64[ns]")
         b["time"] = b.time.astype("datetime64[ns]")
 
+    if "fid" in a:
```

Comment:
Should we get rid of the `ignore_metas` that I added specifically to fix the tests? It might no longer be required.

Reply:
Feel free to takeover this branch to try so, I didn't dare to touch that code ;)